cli/command/formatter/stats.go

```diff
@@ -25,7 +25,9 @@ const (
 
 // StatsEntry represents represents the statistics data collected from a container
 type StatsEntry struct {
+	Container        string
 	Name             string
+	ID               string
```

Comment:
No reason to nit pick when its still in design review @thaJeztah.  be nice to @vdemeester